index.html

```diff
@@ -22,5 +24,44 @@
       </div>
       <a href="">Download</a>
     </nav>
+    <section class="askedQuestion">
+      <div class="askedQuestion_left">
+      
+        <details>
+          <summary>Frequently Asked Question 1<span>&#8250;</span></summary>
+            Morbi vehicula acru et pellentesque tincidunt.
+            Nunt ligula nulla, lobortis a elementum non, vulputate ut acru.
+            Aliquam erat volutpat. Nullam lacinia felis.
+       </details>
+       <details>
+         <summary>Frequently Asked Question 2<span>&#8250;</span></summary>
+           Morbi vehicula acru et pellentesque tincidunt.
+           Nunt ligula nulla, lobortis a elementum non, vulputate ut acru.
+           Aliquam erat volutpat. Nullam lacinia felis.
+       </details>
+       <details>
+         <summary>Frequently Asked Question 3<span>&#8250;</span></summary>
+           Morbi vehicula acru et pellentesque tincidunt.
+           Nunt ligula nulla, lobortis a elementum non, vulputate ut acru.
+           Aliquam erat volutpat. Nullam lacinia felis.
+       </details>
+       <details>
+         <summary>Frequently Asked Question 4<span>&#8250;</span></summary>
+           Morbi vehicula acru et pellentesque tincidunt.
+           Nunt ligula nulla, lobortis a elementum non, vulputate ut acru.
+           Aliquam erat volutpat. Nullam lacinia felis.
+       </details>
+       <details>
+         <summary>Frequently Asked Question 5<span>&#8250;</span></summary>
+           Morbi vehicula acru et pellentesque tincidunt.
+           Nunt ligula nulla, lobortis a elementum non, vulputate ut acru.
+           Aliquam erat volutpat. Nullam lacinia felis.
+      </details>
+     </div>
+     <div class="askedQuestion_right">
+        <img src="assets/img/about.jpeg" alt="Image of connected databases">
```

Comment:
can remove 'image of ' from the alt description since alt description is in an `<img>` tag so it's already conveyed it's about an image
→ `alt="connected databases"`